commit cfbe861506e2dc3250ac99dc45bb3d1ac60f4857 removed the definition of
monitor_flash_len from the eNET which was not picked up due to extensive
use of the SRAM configuration target for testing

Signed-off-by: Graeme Russ <graeme.russ@gmail.com>
---
 board/eNET/eNET.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/board/eNET/eNET.c b/board/eNET/eNET.c
index dd0ce54..2a5636c 100644
--- a/board/eNET/eNET.c
+++ b/board/eNET/eNET.c
@@ -35,6 +35,8 @@
 
 DECLARE_GLOBAL_DATA_PTR;
 
+unsigned long monitor_flash_len = CONFIG_SYS_MONITOR_LEN;
+
 static void enet_timer_isr(void);
 static void enet_toggle_run_led(void);
 static void enet_setup_pars(void);
-- 
1.7.1.422.g049e9

_______________________________________________
U-Boot mailing list
U-Boot@lists.denx.de
http://lists.denx.de/mailman/listinfo/u-boot
